gen_code_can_common.c

```diff
@@ -0,0 +1,820 @@
+
+
+}
+
+bool get_sensor_altitude(const can_msg_t *msg,
+                        uint32_t altitude, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!altitude) { return false; } 
+
+
+}
+
+bool get_sensor_acc(const can_msg_t *msg,
+                        uint16_t x, 
+                           uint16_t y, 
+                           uint16_t z, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!x) { return false; } 
+    if (!y) { return false; } 
+    if (!z) { return false; } 
+
+
+}
+
+bool get_sensor_acc2(const can_msg_t *msg,
+                        uint16_t x, 
+                           uint16_t y, 
+                           uint16_t z, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!x) { return false; } 
+    if (!y) { return false; } 
+    if (!z) { return false; } 
+
+
+}
+
+bool get_sensor_gyro(const can_msg_t *msg,
+                        uint16_t x, 
+                           uint16_t y, 
+                           uint16_t z, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!x) { return false; } 
+    if (!y) { return false; } 
+    if (!z) { return false; } 
+
+
+}
+
+bool get_sensor_mag(const can_msg_t *msg,
+                        uint16_t x, 
+                           uint16_t y, 
+                           uint16_t z, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!x) { return false; } 
+    if (!y) { return false; } 
+    if (!z) { return false; } 
+
+
+}
+
+bool get_sensor_analog(const can_msg_t *msg,
+                        uint16_t value, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!value) { return false; } 
+
+
+}
+
+bool get_gps_timestamp(const can_msg_t *msg,
+                        uint8_t hrs, 
+                           uint8_t mins, 
+                           uint8_t secs, 
+                           uint8_t dsecs, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!hrs) { return false; } 
+    if (!mins) { return false; } 
+    if (!secs) { return false; } 
+    if (!dsecs) { return false; } 
+
+
+}
+
+bool get_gps_latitude(const can_msg_t *msg,
+                        uint8_t degs, 
+                           uint8_t mins, 
+                           uint16_t dmins, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!degs) { return false; } 
+    if (!mins) { return false; } 
+    if (!dmins) { return false; } 
+
+
+}
+
+bool get_gps_longitude(const can_msg_t *msg,
+                        uint8_t degs, 
+                           uint8_t mins, 
+                           uint16_t dmins, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!degs) { return false; } 
+    if (!mins) { return false; } 
+    if (!dmins) { return false; } 
+
+
+}
+
+bool get_gps_altitude(const can_msg_t *msg,
+                        uint16_t altitude, 
+                           uint8_t daltitude, 
+)
+
+{
+    if (!msg){ return false; }
+    if (!altitude) { return false; } 
+    if (!daltitude) { return false; } 
+
+
+}
+
+bool get_gps_info(const can_msg_t *msg,
```

Comment:
params: `num_sats` instead of `num_sat`
missing:
```
    if (get_message_type(msg) != MSG_GPS_INFO) { return false; }

    *num_sat = msg->data[3];
    *quality = msg->data[4];

    return true;
```